packages/design-system-scripts/gulp/docs/generatePages/generateReactExample.js

```diff
@@ -35,6 +35,7 @@ function generateReactExample(page, docsPath, sourceDir, { typescript, rootPath
       const head = `
         <title>Example: ${page.reference}</title>
         <link rel="stylesheet" href="/${path.join(rootPath, 'example.css')}" />
+        ${core ? analytics() : ''}
```

Comment:
This is purely for testing right?

Reply:
I thinking we monitor how ga tracks this for now. If users click `New tab` to open the iframe, I think it'd be good to track the page usage.

Reply:
Hm, I'm not sure if thats necessary. They are just code examples, we dont need to track users using them.

Reply:
Isnt it possible to manually test the tealium events by simply pasting the tealium script on the HelpDrawer code example page?

Reply:
Ok, will check that out.